### PR TITLE
fix #279210: cannot change properties of multiple notes in piano roll editor

### PR DESCRIPTION
**Pull request by Howard-C**
Resolves: https://musescore.org/node/279210.

This allows properties of multiple notes be edited at once, the same way properties of multiple elements can be changed in the inspector.

### CODE REVIEW

File: mscore/pianoroll.cpp [hunk 1]
```diff
@@ -312,15 +309,15 @@ PianorollEditor::~PianorollEditor()
       }
 
 //---------------------------------------------------------
-//   focucOnElement
+//   focusOnElement
```
Comment (Jojo-Schmitz) [inline mscore/pianoroll.cpp]: Whatis the difference? I don't see any

Comment (Spire42) [reply mscore/pianoroll.cpp]: Fixed a typo in the comment (“focuc” to “focus”).

Comment (Jojo-Schmitz) [reply mscore/pianoroll.cpp]: Outsch, I've been looking really hard, but didn't spot it

File: mscore/pianoroll/pianoroll.cpp [hunk 2]
```diff
@@ -773,18 +789,22 @@ void PianorollEditor::posChanged(POS p, unsigned tick)
...
-      NoteEvent* event = item->getTweakNoteEvent();
-      if (!event || event->ontime() == val)
+      if (!items.size())
             return;
 
-      NoteEvent ne = *event;
-      ne.setOntime(val);
       _score->startCmd();
-      _score->undo(new ChangeNoteEvent(note, event, ne));
+      for (int i = 0; i < items.size(); i++) {
```
Comment (anatoly-os) [inline mscore/pianoroll/pianoroll.cpp:796]: Would you mind extracting this code to a separate method? I see the code duplication in `PianorollEditor::tickLenChanged` method.

**Comment by anatoly-os** [comment]
@blackears could you please review the changes?

File: mscore/pianoroll/pianoroll.cpp (hunk 2 above)
Comment (Howard-C) [reply mscore/pianoroll/pianoroll.cpp:796]: Is it really necessary? I see function differences such as `event->ontime()` and `event->len()`. Even if I use a template function, it's hard to make these member functions a kind of variables.

**Pull request merged**